docs/styles/styles.scss

```diff
@@ -284,3 +298,15 @@ footer {
     max-width: var(--content-max-width);
   }
 }
+
+.sr-only {
```

Comment:
Just curious, why do we need to re-implement visually hidden styles here? Just want to make sure I'm understanding this class correctly!

Reply:
So the markdown-it plugin will generate the link with an accessible text. But that text is only for the screenreader. So when this link is focused, it allows to be actually read nicely. Visually it only shows the symbol, but when read, it will be read as 'Section titled "headline"'. That is why the `.sr-only` class is here. I was looking for another implementation of it but I couldn't find it! Sorry if it exists already.

Reply:
Ah got it! No don't worry, this is all news to me. Just wasn't sure how it all worked 👍